### DISCUSSION TITLE
Bitbucket Cloud does support `automergeStrategy=rebase` as of sep 30, 2024

### DISCUSSION THREAD

**Discussion by jmartens** [renovatebot/renovate#33164] [Suggest an Idea]
### Tell us more.

In the renovate docs it is stated that [BitBucket does not support `automergeStrategy=rebase`](https://docs.renovatebot.com/modules/platform/bitbucket/#unsupported-platform-featuresconcepts), with a reference to https://jira.atlassian.com/browse/BCLOUD-16610 but that same bug mentions:

> Latest update from Atlassian 2024-09-30
> 
> We've added support for the following additional merge strategies:
> 
> Rebase + fast forward ([BCLOUD-16610](https://jira.atlassian.com/browse/BCLOUD-16610))
> Rebase + merge ([BCLOUD-17881](https://jira.atlassian.com/browse/BCLOUD-17881), [BCLOUD-19186](https://jira.atlassian.com/browse/BCLOUD-19186))
> Squash, fast forward only ([BCLOUD-16338](https://jira.atlassian.com/browse/BCLOUD-16338))
> It's also now possible to sync a feature branch via rebase ([BCLOUD-6202](https://jira.atlassian.com/browse/BCLOUD-6202)).
> 
> https://bitbucket.org/blog/branch-sync-merge-rebase

**Comment by viceice** [comment]
feel free to send a PR to update 🤗
